layouts/millennium_dstu2_sidebar.html

```diff
@@ -1,4 +1,7 @@
 <div id="js-sidebar" class="sidebar-shell">
+  <div class="expand-all">
+    <a href="javascript:;">+ Expand All</a>
```

Comment:
is the `javascript:;` portion needed?

Reply:
Updated here: https://github.com/cerner/fhir.cerner.com/pull/431/commits/1df23032b655b26ff3cbf71bd84e40e0061c9499 https://github.com/cerner/fhir.cerner.com/pull/431/commits/0baf63e3ea3633fa55a34e5b728142f6662b959c#diff-46bd8686b23677fb5c466a0f71fa66fdR105